ecommerce/views.py

```diff
@@ -40,3 +41,21 @@ def update_profile(request):
         user_form = UserForm(instance=request.user)
         profile_form = ProfileForm(instance=request.user.profile)
     return render(request, 'ecommerce/profile.html', {'user_form': user_form,'profile_form': profile_form})
+
+@login_required
+def add_to_wishlist(request):
+    if request.method == "POST":
+        favorite = FavoriteProduct()
+        product_id = request.POST.get("product_pk")
+        favorite.user = User.objects.get(id=request.user.id)
```

Comment:
dùng trực tiếp current user của request luôn đi em
```suggestion
        favorite.user = request.user
```